libopenage/renderer/opengl/renderer.h

```diff
@@ -63,6 +63,12 @@ class GlRenderer final : public Renderer {
 	/// Optimize the render pass by reordering stuff
 	static void optimize(const std::shared_ptr<GlRenderPass> &pass);
 
+	void setupStencilWrite();
+
+	void setupStencilTest();
+
+	void disableStencilTest();
```

Comment:
These functions should use snake case naming, e.g. `setup_stencil_write`, to adhere to our code style conventions.